### PR TITLE
Remove MGDoFHandler references

### PR DESCRIPTION
**Pull request by drwells**
I found these while working on #1907.

### CODE REVIEW

File: include/deal.II/dofs/block_info.h [hunk 1]
```diff
@@ -64,16 +64,21 @@ namespace hp
  * The most common usage for this object is initializing vectors as in the
  * following code:
  *
- * <code> MGDofHandler<dim> dof_handler(triangulation);
- * dof_handler.distribute_dofs(fesystem);
+ * <code>
+ * DoFHandler<dim> dof_handler(triangulation);
+ * dof_handler.distribute_dofs(fe_system);
```
Comment (bangerth) [inline include/deal.II/dofs/block_info.h]: After the removal of MGDoFHandler, you actually need to add the line
```
  dof_handler.distribute_mg_dofs(fe_system);
```
to ensure that it actually builds MG dofs as well.

**Comment by bangerth** [comment]
OK if you add the one missing line of code.

**Comment by drwells** [comment]
Nice catch.

Just checking: when you say 'OK if', you mean that I should just fix things and merge myself, right?

**Comment by bangerth** [comment]
Yes, please merge yourself.

**Pull request merged**